packages/nimbus/src/components/text-input/text-input.dev.mdx

```diff
@@ -379,11 +379,12 @@ These examples demonstrate how to test your implementation when using TextInput
 
 Verify the component renders with expected elements:
 
+<!-- test:validate -->
```

Comment:
This is the opening tag for code blocks in the docs which allows the script to extract them. We would need to apply this to all current and future engineering docs